test/tracedump.ml

```diff
@@ -22,6 +23,7 @@ let print_error = function
   | `Ambiguous_uri -> prerr_string "More than one provider for a given URI"
 
 let main uri =
+  let _ = Plugins.load() in
```

Comment:
According to your recommends I have moved tracedump utility to bap repository and made additional pull requests to bap repository and opam-repository